confidant_client/cli.py

```diff
@@ -474,6 +470,18 @@ def _parse_args():
         default=120,
     )
 
+    ca_get_ca_parser = subparsers.add_parser(
+        'get_ca'
+    )
+    ca_get_ca_parser.add_argument(
+        '--ca',
+        required=True,
+        help='The certificate authority to get.',
```

Comment:
should we clarify in the help that this is in an `arn` format?

Reply:
It's actually not an arn format for this. It's the "friendly" name configured by the server.

Reply:
thanks, makes sense after digging into the server side.